site/src/pages/UsersPage/UsersPage.tsx

```diff
@@ -91,19 +74,26 @@ export const UsersPage: FC<{ children?: ReactNode }> = () => {
         status: option?.value,
       }),
   });
-  const authMethods = useQuery({
-    queryKey: ["authMethods"],
-    queryFn: () => {
-      return getAuthMethods();
-    },
-  });
-  // Is loading if
-  // - users are loading or
-  // - the user can edit the users but the roles are loading
+  const authMethodsQuery = useQuery(authMethods());
   const isLoading =
-    usersState.matches("gettingUsers") ||
-    rolesQuery.isLoading ||
-    authMethods.isLoading;
+    usersQuery.isLoading || rolesQuery.isLoading || authMethodsQuery.isLoading;
+  // Suspend
```

Comment:
these comments don't really add much value, and feel like the kind that are prone to eventual drift. maybe just separate each group with a blank line?

Reply:
Ok, I personally like to use comments to separate things sometimes 😁 going to remove them and add blank lines instead